sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/DataSource.scala

```diff
@@ -724,4 +726,37 @@ object DataSource extends Logging {
          """.stripMargin)
     }
   }
+
+  /**
+   * Return all paths represented by the wildcard string.
+   * This will be done in main thread by default while the value of config
+   * `spark.sql.sources.parallelGetGlobbedPath.numThreads` > 0, a local thread
+   * pool will expand the globbed paths.
```

Comment:
Can you describe a note about what's the difference from `InMemoryFileIndex.listLeafFiles`?

Reply:
Thanks, done in 1319cd3.